CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.27.3] - 2024-11-05
+
+### Fixed
+
+- CASMTRIAGE-7445: iSCSI is reporting "SQUASHFS errors" on gamora for unknown reasons
```

Comment:
We probably don't want the CHANGELOG to reference specific system names like this. It should just summarize the problem being fixed.